### PR TITLE
test(#1617): Add test for `AtLogged`

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Add test for `AtLogged`.

Closes: #1617

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@mximp Can you take a look, please?

File: eo-runtime/src/main/java/org/eolang/AtLogged.java [hunk 1]
```diff
@@ -44,14 +44,30 @@ final class AtLogged implements Attr {
      */
     private final String owner;
 
+    /**
+     * Output stream.
+     */
+    private final PrintStream out;
```
Comment (mximp) [inline eo-runtime/src/main/java/org/eolang/AtLogged.java:50]: @volodya-lombrozo why don't we use a logger here like we do in `PhDefault` and `Dataized`?

Comment (volodya-lombrozo) [reply eo-runtime/src/main/java/org/eolang/AtLogged.java:52]: @mximp Yea, I agree, it would be much better. I've added puzzle for that. Check, please.

**Review by mximp — APPROVED** [review]

**Comment by volodya-lombrozo** [comment]
@yegor256 Can you merge, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**